template.tpl

```diff
@@ -767,104 +759,217 @@ ___TEMPLATE_PARAMETERS___
         "newRowButtonText": "Add configuration"
       },
       {
-        "enablingConditions": [
-          {
-            "paramName": "attribution",
-            "type": "EQUALS",
-            "paramValue": true
-          }
-        ],
-        "displayName": "Attribution Options",
-        "name": "attributionOptions",
-        "groupStyle": "ZIPPY_OPEN",
-        "type": "GROUP",
+        "help": "Wether to use client side user agent enrichment. Enable client side user agent to make the changes compatible with previous GTM template version.",
```

Comment:
Maybe explicitly write the version, WRAPPER_VERSION or template SHA version ?

Reply:
```suggestion
        "help": "Whether to use client-side user agent enrichment. Enable client-side user agent to make the changes compatible with the previous GTM template version.",
```